src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
+        complete: arguments[2]
+      };
+    }
+
+    if (observer.next) {
+      this.onValue(observer.next);
```

Comment:
I can definitely port over that code. Here's the thing: If we're going to add a `subscribe` function that matches the `es-observable` spec, then we don't actually need to provide the `toESObservable` method. Should we remove/deprecate that method are part of this PR?

I ask because we'd end up with code duplication if I just copied it.